build/Dockerfile

```diff
@@ -6,36 +6,38 @@ RUN dpkg --add-architecture i386 \
 	&& apt-get update -y \
 	&& apt-get install -y --no-install-recommends ca-certificates curl lib32gcc1 lib32stdc++6 libcurl4-gnutls-dev:i386 \
 	&& apt-get clean \
-	&& rm -rf /var/lib/apt/lists/*
-
-# Install gosu.
```

Comment:
What I'm arguing is: are we going to change the `gosu` layer? I think that we are going to change more the `atp-get` layer. Any way have one or two layers here is not a big deal. I'll remove that commit.